.cassandra-versions

```diff
@@ -1,108 +1,3 @@
-0.3.0
-0.4.0
-0.4.1
-0.4.2
-0.5.0
-0.5.1
-0.6.0
-0.6.1
-0.6.2
-0.6.3
-0.6.4
-0.6.5
-0.6.6
-0.6.7
-0.6.8
-0.6.9
-0.6.10
-0.6.11
-0.6.12
-0.6.13
-0.7.0
-0.7.1
-0.7.2
-0.7.3
-0.7.4
-0.7.5
-0.7.6
-0.7.7
-0.7.8
-0.7.9
-0.7.10
-0.8.0
-0.8.1
-0.8.2
-0.8.3
-0.8.4
-0.8.5
-0.8.6
-0.8.7
-0.8.8
-0.8.9
-0.8.10
-1.0.0
-1.0.1
-1.0.2
-1.0.3
-1.0.4
-1.0.5
-1.0.6
-1.0.7
-1.0.8
-1.0.9
-1.0.10
-1.0.11
-1.0.12
-1.1.0
-1.1.1
-1.1.2
-1.1.3
-1.1.4
-1.1.5
-1.1.6
-1.1.7
-1.1.8
-1.1.9
-1.1.10
-1.1.11
-1.1.12
-1.2.0
-1.2.1
-1.2.2
-1.2.3
-1.2.4
-1.2.5
-1.2.6
-1.2.7
-1.2.8
-1.2.9
-1.2.10
-1.2.11
-1.2.12
-1.2.13
-1.2.14
-1.2.15
-1.2.16
-1.2.17
-1.2.18
-1.2.19
-2.0.0
-2.0.1
-2.0.2
-2.0.3
-2.0.4
-2.0.5
-2.0.6
-2.0.7
-2.0.8
-2.0.9
-2.0.10
-2.0.11
-2.0.12
-2.0.13
-2.0.14
-2.0.15
-2.0.16
```

Comment:
No problem! I probably should’ve written the task to put a comment at the top of the file—not the first time this confusion has occurred